base/utils/data/standard_vars.csv

```diff
@@ -104,7 +105,8 @@ Root carbon content, optionally by size class; alternatively specify fine_ and c
 "PAR","surface_downwelling_photosynthetic_photon_flux_in_air","mol m-2 s-1","Photosynthetically Active Radiation","Driver","real","lon","lat","time",NA,"Photosynthetically Active Radiation"
 "precipf",NA,"kg m-2 s-1","Precipitation","Driver","real","lon","lat","time",NA,"The per unit area and time precipitation representing the sum of convective rainfall, stratiform rainfall, and snowfall"
 "BA",NA,"m2 ha-1","Basal area","Diversity","real","lon","lat","time","pft","Basal area by PFT"
-"Dens",NA,"1 ha-1","Stem Density","Diversity","real","lon","lat","time","pft","Stem Density dy PFT"
+"Dens",NA,"1 ha-1","Stem Density","Diversity","real","lon","lat","time","pft","Stem Density by PFT"
+"DBH",NA,"cm","Diameter at Breast Height","Diversity","real","dbhsize","time","pft","NA","DBH by PFT"
```

Comment:
This variable is ambiguous. If DBH has a size class dimension that is cm, what numbers are going into `DBH` itself that could also have dimensions of cm?

Reply:
hmm `dbhsize` is something like `c(10,20,30,40,50)` referring to the break points which are in cm, then DBH is the average of trees that are falling into that bin, so it's also cm.

btw, actually the read.output code is currently not compatible with anything other than `dbh_breaks = 0` and I don't think anyone other than me will use it in the near future but I put it there any way to prepare for future use (and because Ryan already implemented it). I will revisit this idea when pushing the ESM code

Reply:
Why is the average DBH within a DBH bin something interesting/useful to output? What does this tell you? I don't think I've ever seen anyone report this before. Reporting stem density by DBH bin, by contrast, does seem potentially useful because that would give you a size distribution. Growth and mortality by size would also be useful. But I think you need a better justification for why we need size by size.

Reply:
IIRC we were rebinning to common DBH bins to be able to assimilate Jim's growth estimates by size class.. can discuss tomorrow